rotkehlchen/chain/ethereum/modules/ens/decoder.py

```diff
@@ -202,18 +202,19 @@ def _decode_ens_registry_with_fallback_event(
                 balance=Balance(),
                 location_label=transaction.from_address,
                 notes=notes,
-                counterparty=CPT_ENS,
+                protocol=CPT_ENS,
+                address=cast(ChecksumEvmAddress, transaction.to_address),  # transaction.to_address is not None here  # noqa: E501
```

Comment:
Is ugly that you had to do this in so many places but right now I can't think of a better solution :thinking:

Reply:
Me too